src/test/kotlin/uk/gov/justice/digital/hmpps/suicideriskformapi/integration/health/HealthCheckTest.kt

```diff
@@ -5,18 +5,18 @@ import uk.gov.justice.digital.hmpps.suicideriskformapi.integration.IntegrationTe
 
 class HealthCheckTest : IntegrationTestBase() {
 
-  @Test
-  fun `Health page reports ok`() {
-    stubPingWithResponse(200)
-
-    webTestClient.get()
-      .uri("/health")
-      .exchange()
-      .expectStatus()
-      .isOk
-      .expectBody()
-      .jsonPath("status").isEqualTo("UP")
-  }
+//  @Test
+//  fun `Health page reports ok`() {
```

Comment:
Why is this commented out, is it failing?  Sorry, you've probably already told me this 😅